multivac/gather_data.py

```diff
@@ -105,6 +106,16 @@ def github_time_to_unix(time: str) -> float:
     return time_to_unix
 
 
+# https://github.com/tarantool/tarantool/tree/master/.github/workflows
+os_matcher = re.compile(r"[a-z]+_[0-9]+(_[0-9]+)?")
```

Comment:
Usually, such constants should be in uppercase.

Reply:
fixed